kube_apiserver_metrics/tests/test_kube_apiserver_metrics_1_25.py

```diff
@@ -0,0 +1,115 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+# stdlib
+import os
+
+import mock
+import pytest
+
+from datadog_checks.kube_apiserver_metrics import KubeAPIServerMetricsCheck
+
+customtag = "custom:tag"
+
+instance = {
+    'prometheus_url': 'https://localhost:443/metrics',
+    'bearer_token_auth': 'false',
+    'tags': [customtag],
+}
+
+
+@pytest.fixture()
+def mock_get():
+    f_name = os.path.join(os.path.dirname(__file__), 'fixtures', 'metrics_1.24.0.txt')
+    with open(f_name, 'r') as f:
+        text_data = f.read()
+    with mock.patch(
+        'requests.get',
+        return_value=mock.MagicMock(
+            status_code=200,
+            iter_lines=lambda **kwargs: text_data.split("\n"),
+            headers={'Content-Type': "text/plain", 'Authorization': "Bearer XXX"},
+        ),
+    ):
+        yield
+
+
+class TestKubeAPIServerMetrics:
```

Comment:
Hmm, are you aware of #14147? This test looks exactly the same as the one in that PR... my guess is there will be merge conflicts, it may make sense to resolve them early.